c2corg_images/__init__.py

```diff
@@ -3,15 +3,19 @@
 from pyramid.events import NewRequest
 import os
 
+
 # See http://docs.wand-py.org/en/0.4.1/guide/resizecrop.html
 # Max 800, keep aspect ratio
 # Max 120x120 don't keep ratio
 THUMBNAIL_CONFIGS = [
-    {'suffix': 'BI', 'convert': ['-resize', '800x800>']},
-    {'suffix': 'MI', 'convert': ['-resize', '250x250>']},
+    {'suffix': 'BI', 'convert': ['-resize', '800x800>',
+                                 '-quality', '80']},
```

Comment:
For an example : 
80 => 182ko
90 => 262ko with visible quality improvement.

The default value is 92, from the ImageMagic docs. The default value in most softwares is 90.

IMHO, we should use 90.